config/jobs/cert-manager/cert-manager-periodics.yaml

```
@@ -512,7 +512,7 @@ periodics:
     containers:
     - image: eu.gcr.io/jetstack-build-infra-images/golang-aws@sha256:1f330e4c9552ca383d157067b73fb0e090b64b0777939fd59e58b60e06020d66
       args:
-      - -c
+      - bash -c
```

bash and -c have to be separate times in the list
```suggestion
      - bash
      - -c
```